app/javascript/components/student/mentoring-session/mentoring-request/MentoringRequestForm.tsx

```diff
@@ -110,7 +110,7 @@ export const MentoringRequestForm = ({
           </label>
           <p id="request-mentoring-form-solution-description">
             Give your mentor a starting point for the conversation. This will be
-            your first comment on during the session.
+            your first comment during the session (markdown is permitted).
```

Comment:
One tiny suggestion:

```suggestion
            your first comment during the session. Markdown is permitted.
```